companies/zoom.md

```diff
@@ -0,0 +1,28 @@
+---
+fortune_rank:
+name: Zoom Video Communications, Inc.
+slug: zoom
+blm_statements:
+  - url:
+    date_posted:
```

Comment:
```suggestion
    date_posted: 2020-05-30
```

Reply:
If I'm adding the link to line 6 I will add the date.